repository-hbase/src/main/scala/hbase/util/HBaseConfig.scala

```diff
@@ -0,0 +1,24 @@
+package hbase.util
+
+import org.apache.hadoop.hbase.TableName
+import com.typesafe.config.{ Config, ConfigFactory }
+
+/**
+ * HBaseConfig
+ * ----------------
+ * Author: haqa
+ * Date: 06 December 2017 - 16:43
+ * Copyright (c) 2017  Office for National Statistics
+ */
+object HBaseConfig {
+  lazy val config: Config = ConfigFactory.load().getConfig("hbase")
```

Comment:
As above, this isn't good practice since we should inject the `Config` / `Configuration` instances. However, this will be improved in my PR so we can wait for that.